lib/conjur/conjur_config.rb

```diff
@@ -23,6 +23,7 @@ class ConjurConfig < Anyway::Config
     attr_config(
       # Read TRUSTED_PROXIES before default to maintain backwards compatibility
       trusted_proxies: (ENV['TRUSTED_PROXIES'] || []),
+      api_resource_list_limit: 0, # The maximum number of results for listing requests. The default value is 0 and means that there is no limit.
```

Comment:
Was there a reason for leaving the `_max` off of the suggestion? Just curious.

Also, please place the comment on the line above the config definition. See the `trusted_proxies` example above.

Reply:
updated the comment,
seems to me redundant to use both `max` and `limit` wdyt?

Reply:
`limit` is the name of the parameter. What we are configuring is the maximum value allowed in the parameter.

Reply:
updated